ts/src/base/Exchange.ts

```diff
@@ -2316,6 +2316,36 @@ export default class Exchange {
         throw new NotSupported (this.id + ' fetchMarginModes () is not supported yet');
     }
 
+    async retryMethod (methodName: string, args: any[], retries: Int = undefined) {
```

Comment:
how is this better than fetchObSafe? dynamic calls should be avoided, this is a pain in languages like c# when the arguments mismatch....

Reply:
1) because it can be used for any method in future
2) because more expressive, fetchOrderBookSafe is not clear what it does
3) but we already have dynamic calls https://github.com/ccxt/ccxt/blob/master/ts/src/base/Exchange.ts#L6465 , https://github.com/ccxt/ccxt/blob/master/ts/src/base/Exchange.ts#L6525 and many other places

well idk, i just thought it would have been slight progress, not critical though

Reply:
@carlosmiei wdyt about this? if not, plz feel free to close & delete branch